cdk/lib/payment-api.ts

```diff
@@ -145,6 +145,13 @@ export class PaymentApi extends GuStack {
                 ? [`arn:aws:sqs:${this.region}:${this.account}:soft-opt-in-consent-setter-queue-PROD`,]
                 : [`arn:aws:sqs:${this.region}:${this.account}:soft-opt-in-consent-setter-queue-DEV`,],
           }),
+          new GuAllowPolicy(this, "SingleContributionRecordSqsMessages", {
+            actions: ["sqs:GetQueueUrl", "sqs:SendMessage"],
+            resources:
+              this.stage === "PROD"
+                ? [`arn:aws:sqs:${this.region}:${this.account}:single-contribution-record-queue-PROD`,]
+                : [`arn:aws:sqs:${this.region}:${this.account}:single-contribution-record-queue-DEV`,],
```

Comment:
@rupertbates should this be referring to PROD and **CODE**, not DEV? Andrea is just copying the  SoftOptInsSqsMessages from above.

Reply:
Yes should be CODE

Reply:
Thank you. Updated.